### ISSUE TITLE
Accessibility issue

### ISSUE THREAD

**Issue by ThePix** [textadventures/quest#1047]
copied from here:
http://textadventures.co.uk/forum/quest/topic/lhr-txio90icnou7j2cwlq/quest-desktop-accessibility-issues

Issue 1: unlabeled text fields
As I already mentioned in the Quest 5.8-topic, many, if not all textfields are unlabeled or just labeled using the title Attribute. Voice may or may not announce this title (depends on the Situation) but my Braille Display doesn't Show it - at least not with Jaws running (NVDA might - can't check yet). Using Labels would bebetter.

Issue 2: Context often hard to understand.
As I can only navigate effectively using TAB and ctrl-TAB, I probably miss Labels or captions for certain things. Lets take the scripts section of the first room object - you can add various scripts for different purposes, however in the Desktop Version I can't tell apart which "add new script"-button is for what. I Need to compare this structure to the web Client to figure this out (because in the web Client everything is properly labeled).

Issue 3: Unlabeled Buttons.
The move up, move down, cut, copy, delete... Buttons are all unlabeled and thus get only called "button". This isn't ideal, as I can memorize the order of the Buttons to select the appropriate Thing, but it still could lead to accidents Messing up my work.

EDIT:

Issue 4: Attribute tab somehow leads to issues.
This is what confuses me the most. In game/Features I ticked the Money Checkbox, thus automatically adding a Money Attribute. If I want to view it, first a text field with $! gets displayed (turns out this is the money type, apparently), then I press tab and Key gets displayed - and the application somehow jams. I'm unable to execute any further Action until I press alt-tab twice, thus leaving and re entering the application (I land back on the page selector where i started).
Hence, I'm unable to create or Change attributes. As this is not part of the web Editor, I'm unable to do this any other way, either. This issue (or similar issues) appear everywhere where I theoretically could adjust attributes. Only exception is where attributes with values already exist, but even there I can't add, just view and alter, plus I'm unable to get back out of the attributes section and onto the game/rom/player... tree.
Issue 5: Code view inaccessible.
That basically says it all. It's utterly inaccessible, I can't view anything, interact with it etc.

Issue 6: running game inaccessible.
The text of the game gets read by voice once at the beginning, but I can't read it on the braille display nor interact with it.

EDIT #2:
As expected, NVDA is a bit better, especially when it comes to attribute navigation (EDIT: sometimes). But I still can't add attributes, can't leave the attribute tab, textfields and buttons are unlabeled, code view is inaccessible.

EDIT #3:
Running the game is largely inaccessible with NVDA as well - I can only see the text field, not the page content.

**Comment by PhoenixTilea** [comment]
Hi. I have also run into some of these issues with the desktop version. As I am a coder, I'm going to take a shot at fixing some of this, but I'm relatively green with C#, so I can't promise anything at present.

**Comment by alexwarren** [comment]
Superceded by #1294
